vendor/bat-native-ads/src/bat/ads/internal/ads_impl.cc

```diff
@@ -788,26 +791,27 @@ std::string AdsImpl::ClassifyPage(
     const std::string& url,
     const std::string& content) {
   DCHECK(user_model_);
-  auto page_score = user_model_->ClassifyPage(content);
 
-  auto winning_category = GetWinningCategory(page_score);
+  const std::map<std::string, double> page_classification =
```

Comment:
could we use `using CategoryScoreList = std::map<std::string, double>`. Or even more concrete `CategoryLikelihoodList` or `CategoryProbabilityList`?

Reply:
Agreed to go with `CategoryProbabilityList`